core/consensus/src/status.rs

```diff
@@ -67,45 +67,45 @@ impl StatusAgent {
 
 #[derive(Serialize, Deserialize, Clone, Debug, Display)]
 #[display(
-    fmt = "current_height {}, exec height {}, current_hash {:?}, latest_commited_state_root {:?} list state root {:?}, list receipt root {:?}, list confirm root {:?}, list cycle used {:?}, logs bloom {:?}",
+    fmt = "current_height {}, exec height {}, current_hash {:?}, latest_committed_state_root {:?} list state root {:?}, list receipt root {:?}, list confirm root {:?}, list cycle used {:?}, logs bloom {:?}",
     current_height,
     exec_height,
     current_hash,
-    latest_commited_state_root,
+    latest_committed_state_root,
     list_state_root,
     list_receipt_root,
     list_confirm_root,
     list_cycles_used,
     "list_logs_bloom.iter().map(|bloom| bloom.to_low_u64_be()).collect::<Vec<_>>()"
 )]
 pub struct CurrentConsensusStatus {
-    pub cycles_price:               u64,
-    pub cycles_limit:               u64,
-    pub current_height:             u64,
-    pub exec_height:                u64,
-    pub current_hash:               Hash,
-    pub latest_commited_state_root: MerkleRoot,
-    pub list_logs_bloom:            Vec<Bloom>,
-    pub list_confirm_root:          Vec<MerkleRoot>,
-    pub list_state_root:            Vec<MerkleRoot>,
-    pub list_receipt_root:          Vec<MerkleRoot>,
-    pub list_cycles_used:           Vec<u64>,
-    pub current_proof:              Proof,
-    pub validators:                 Vec<Validator>,
-    pub consensus_interval:         u64,
-    pub propose_ratio:              u64,
-    pub prevote_ratio:              u64,
-    pub precommit_ratio:            u64,
-    pub brake_ratio:                u64,
-    pub tx_num_limit:               u64,
-    pub max_tx_size:                u64,
-}
+    pub cycles_price:                u64, // metadata
+    pub cycles_limit:                u64, // metadata
+    pub current_height:              u64, // latest consented height
```

Comment:
latest_committed_height

Reply:
no, consented and consentign should be more clear

Reply:
changed